server/src/test/java/org/opensearch/index/translog/transfer/BaseTranslogTransferManagerTests.java

```diff
@@ -184,11 +203,11 @@ public void onUploadFailed(TransferSnapshot transferSnapshot, Exception ex) {
                 translogTransferFailed.incrementAndGet();
             }
         }));
-        assertEquals(4, fileTransferSucceeded.get());
+        assertEquals(2, fileTransferSucceeded.get());
```

Comment:
pls correct me if my understanding is incorrect. So, the count has halved due to us combined tlog and ckp as one snapshot per generation and the listener gets invoked once per generation?

Reply:
right..Now we are updating this variable per one generation.